### PR TITLE
Sophos app docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create the app docs for Sophos.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Create Public Doc for Sophos App](https://sumologic.atlassian.net/browse/DOCS-418)

### CODE REVIEW

File: blog-service/2024-09-02-apps.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - apps
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of our new Sophos app for Sumo Logic. This app leverages the Sumo Logic Cloud-to-Cloud Sophos source to collect the alerts and events logs using the [Sophos Central APIs](https://developer.sophos.com/), thus helping you with robust security monitoring and threat detection capabilities.
+
+Check out our technical documentation [here](/docs/integrations/saas-cloud/sophos/) to learn how to set up and use the Sophos app for Sumo Logic.
```
Comment (kimsauce) [inline blog-service/2024-09-02-apps.md:15]:
```suggestion
We're excited to introduce the new Sophos App for Sumo Logic. This app leverages the Sumo Logic Cloud-to-Cloud Sophos source to collect alerts and event logs through the [Sophos Central APIs](https://developer.sophos.com/), providing you with robust security monitoring and threat detection capabilities.

Explore our technical documentation [here](https://github.com/docs/integrations/saas-cloud/sophos/) to learn how to set up and use the Sophos app for Sumo Logic.
```

File: docs/integrations/saas-cloud/sophos.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+---
+id: sophos
+title: Sophos
+sidebar_label: Sophos
+description: The Sumo Logic app for Sophos offers robust security monitoring and threat detection capabilities by analyzing your organization's Sophos Alerts and Events.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sophos.md:5]:
```suggestion
description: The Sumo Logic app for Sophos offers robust security monitoring and threat detection capabilities by analyzing your organization's Sophos alerts and events.
```

File: docs/integrations/saas-cloud/sophos.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+<img src={useBaseUrl('img/send-data/sophos.jpeg')} alt="icon" width="40"/>
+
+The Sumo Logic app for Sophos provides robust security monitoring and threat detection capabilities, leveraging detailed data from Sophos security solutions. This app includes pre-configured dashboards that offer deep insights into security events and alerts, enabling security teams to quickly identify, analyze, and respond to potential threats. This app provides a comprehensive view of all security events, categorized by severity and type, with visual tools such as geo-location mapping and event timelines to help prioritize incident response. Additionally, it focuses on alert severity, sources, and affected products, highlighting trends and common issues that may require immediate attention. By analyzing the frequency and type of alerts, security teams can track evolving threats and enhance their defence strategies. This app also includes a real-time recent alerts section, allowing for quick assessment and response to active threats. Overall, the Sumo Logic app for Sophos equips organizations with the necessary tools to strengthen their security posture and protect against sophisticated cyber threats.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sophos.md:12]:
```suggestion
The Sumo Logic app for Sophos delivers robust security monitoring and threat detection capabilities by harnessing detailed data from Sophos security solutions. This app features pre-configured dashboards that provide deep insights into security events and alerts, enabling security teams to quickly identify, analyze, and respond to potential threats.

With a comprehensive view of all security events, categorized by severity and type, the app offers visual tools such as geo-location mapping and event timelines to help prioritize incident response. It also emphasizes alert severity, sources, and affected products, highlighting trends and common issues that may require immediate attention.

By analyzing the frequency and types of alerts, security teams can track evolving threats and refine their defense strategies. The app also includes a real-time recent alerts section, allowing for rapid assessment and response to active threats.

Overall, the Sumo Logic app for Sophos equips organizations with the essential tools to strengthen their security posture and defend against sophisticated cyber threats.
```

File: docs/integrations/saas-cloud/sophos.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition | 
+|:--|:--|:--|:--|
+| `Sophos - Blocklisted Alerts` | This alert is fired when blocklisted alerts are generated, signaling a critical security event.
+ | Critical | Count > 0 |
+| `Sophos - High Severity Alerts` | This alert is fired upon generation of high-severity alerts, indicating significant security threats.
+ | Critical | Count > 0 | 
+| `Sophos - High Severity Events` | This alert is fired when high-severity events occur, highlighting major incidents that require immediate attention.
+| Critical | Count > 0 |
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/sophos.md:132]: it looks like the formatting is a bit off here

Screenshot 2024-08-30 at 1 12 10 AM

**Review by kimsauce — CHANGES_REQUESTED** [review]
had a few changes - good job!

File: docs/integrations/saas-cloud/sophos.md (hunk 4 above)
Comment (JV0812) [reply docs/integrations/saas-cloud/sophos.md]: Thanks for identifying this @kimsauce. I have fixed the comments now.

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Merging this PR at the request of @ankitgoelcmu.

**Pull request merged**